scootapi/scoot.thrift

```diff
@@ -3,7 +3,11 @@
 # See github.com/twitter/scoot/README.md for local Thrift prerequisites
 #
 # To Generate files, run from this (github.com/twitter/scoot/scootapi/) directory:
-#     1. thrift --gen go:package_prefix=github.com/twitter/scoot/scootapi/gen-go/,package=scoot,thrift_import=github.com/apache/thrift/lib/go/thrift scoot.thrift
+#     1. thrift -I ../bazel/execution/bazelapi/ --gen go:package_prefix=github.com/twitter/scoot/bazel/gen-go/,package=scoot,thrift_import=github.com/apache/thrift/lib/go/thrift scoot.thrift
```

Comment:
package_prefix

ugh, maybe we should just reference the Makefile target in these comments so we have a canonical source of truth and generation method?

Reply:
yeah, I think that's a good call